### PR TITLE
EmojiPicker & Emoji

### PR DESCRIPTION
**Pull request by Calvein**
EmojiPicker
Emoji

### CODE REVIEW

File: packages/Emoji/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { bool, number, string } from 'prop-types'
+import { Box } from '@welcome-ui/box'
+import Bowser from 'bowser'
+
+// TODO: change path to not use our cdn
+const EMOJI_PATH = 'https://cdn.welcometothejungle.com/common/assets/emojis/'
```
Comment (Calvein) [inline packages/Emoji/index.js:7]: I'll handle it tomorrow 🙈

File: docs/pages/components/emoji-picker.mdx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+```jsx
+function BasicList() {
+  const emojiPicker = useEmojiPicker()
+  const [emoji, setEmoji] = useState()
+
+  return (
+    <>
+      <EmojiPicker.Trigger as={Button} {...emojiPicker}>
+        {emoji ? <Emoji emoji={emoji} /> : "Ouvrer l'Emoji Picker"}
```
Comment (theo-mesnil) [inline docs/pages/components/emoji-picker.mdx:215]:
```suggestion
        {emoji ? <Emoji emoji={emoji} /> : "Ouvrez l'Emoji Picker"}
```

```suggestion
        {emoji ? <Emoji emoji={emoji} /> : "Ouvrir l'Emoji Picker"}
```

File: packages/EmojiPicker/List.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+import { SearchIcon } from '@welcome-ui/icons'
+import { useIsomorphicLayoutEffect } from '@welcome-ui/utils'
+import debounce from 'lodash.debounce'
+import escapeRegExp from 'lodash.escaperegexp'
+import Popper from 'popper.js'
+
+import * as S from './styles'
+import { formatEmojis, getEmojiAlias, HEIGHT, NB_EMOJIS_PER_ROW, ROW_HEIGHT, WIDTH } from './utils'
+
+getEmojiName
```
Comment (mleralec) [inline packages/EmojiPicker/List.js:18]: 🤔

Comment (Calvein) [reply packages/EmojiPicker/List.js:18]: 🙈

**Review by theo-mesnil — APPROVED** [review]
🔥

**Review by mleralec — APPROVED** [review]
👏 🐶

**Pull request merged**